.github/workflows/test_on_push.yaml

```diff
@@ -13,13 +13,19 @@ jobs:
       matrix:
         # We need 1.10.6 here to check that module works with
         # old Tarantool versions that don't have "tuple-keydef"/"tuple-merger" support.
-        tarantool-version: ["1.10.6", "1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
+        tarantool-version: ["1.10.6", "1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7", "2.8"]
+        metrics-version: [""]
         remove-merger: [false]
         include:
           - tarantool-version: "2.7"
             remove-merger: true
+          - tarantool-version: "2.8"
+            metrics-version: "0.1.8"
+          - tarantool-version: "2.8"
+            metrics-version: "0.9.0"
           - tarantool-version: "2.8"
             coveralls: true
+            metrics-version: "0.12.0"
```

Comment:
Why do we need to run tests with three different versions of metrics?

Reply:
I've decided it would be for the best to test pipeline without metrics, with unsupported metrics version, with minimal supported metrics version and the newest metrics version. Maybe it's a bit excessive, but for me it sounds reasonable. In reality, testing with `0.1.8` only adds a single row to coverage and 0.9.0 may differ from 0.12.0 only in terms of metrics performance (and it's a minor ones).

Reply:
Now minimal required is `0.10.0` and not `0.9.0`